app/_data/docs_nav_ce_2.1.x.yml

```diff
@@ -425,159 +425,3 @@
 
```

Comment:
Can you also remove the duplicate non-Kubernetes topics from `docs_nav_kic_1.0.x` and `docs_nav_kic_1.1.x`?